test/e2e/tests/multichain-accounts/multichain-account-list-menu.spec.ts

```diff
@@ -1,61 +1,83 @@
 import SnapSimpleKeyringPage from '../../page-objects/pages/snap-simple-keyring-page';
+import HeaderNavbar from '../../page-objects/pages/header-navbar';
+import HomePage from '../../page-objects/pages/home/homepage';
+import { loginWithBalanceValidation } from '../../page-objects/flows/login.flow';
+import { KNOWN_PUBLIC_KEY_ADDRESSES } from '../../../stub/keyring-bridge';
+import FixtureBuilder from '../../fixtures/fixture-builder';
 import { DAPP_PATH } from '../../constants';
-import { WINDOW_TITLES } from '../../helpers';
+import { WINDOW_TITLES, withFixtures } from '../../helpers';
+import { mockPriceApi } from '../tokens/utils/mocks';
 import { AccountType, withMultichainAccountsDesignEnabled } from './common';
 
 describe('Multichain Accounts - Account tree', function (this: Suite) {
   it('should display basic wallets and accounts', async function () {
     await withMultichainAccountsDesignEnabled(
       {
         title: this.test?.fullTitle(),
-        state: 2,
       },
       async (driver: Driver) => {
         const accountListPage = new AccountListPage(driver);
-        await accountListPage.checkPageIsLoaded({
-          isMultichainAccountsState2Enabled: true,
-        });
+        await accountListPage.checkPageIsLoaded();
 
         // Ensure that wallet information is displayed
         await accountListPage.checkWalletDisplayedInAccountListMenu('Wallet 1');
         await accountListPage.checkWalletDisplayedInAccountListMenu('Wallet 2');
         await accountListPage.checkAddWalletButttonIsDisplayed();
 
-        // BUGBUG
-        // await accountListPage.checkMultichainAccountBalanceDisplayed('$42,500.00');
+        await accountListPage.checkMultichainAccountBalanceDisplayed(
+          '$85,025.00',
+        );
         await accountListPage.checkMultichainAccountBalanceDisplayed('$0.00');
         await accountListPage.checkAccountDisplayedInAccountList('Account 1');
         await accountListPage.checkAccountDisplayedInAccountList('Account 2');
         await accountListPage.checkNumberOfAvailableAccounts(2);
       },
     );
   });
-
   it('should display wallet and accounts for hardware wallet', async function () {
-    await withMultichainAccountsDesignEnabled(
+    await withFixtures(
       {
+        dappOptions: { numberOfTestDapps: 1 },
+        fixtures: new FixtureBuilder()
+          .withLedgerAccount()
+          .withShowFiatTestnetEnabled()
+          .withEnabledNetworks({ eip155: { '0x1': true } })
+          .withConversionRateEnabled()
+          .withPreferencesControllerShowNativeTokenAsMainBalanceDisabled()
+          .build(),
         title: this.test?.fullTitle(),
-        accountType: AccountType.HardwareWallet,
-        state: 2,
+        testSpecificMock: async (mockServer: Mockttp) => {
+          await mockSnapSimpleKeyringAndSite(mockServer);
+          return [await mockPriceApi(mockServer)];
```

Comment:
Any reason why we're putting this in an array? 🤔 The other usage of this function usually just return `return await mock...` or they don't event return anything.